README.rst

```diff
@@ -87,155 +129,156 @@ must be present and contain a dict with Zyte API parameters:
             )
 
         def parse(self, response):
-            yield {"URL": response.url, "HTML": response.body}
-
             print(response.raw_api_response)
             # {
             #     'url': 'https://quotes.toscrape.com/',
             #     'statusCode': 200,
-            #     'browserHtml': '<html> ... </html>',
+            #     'browserHtml': '<html>…</html>',
             # }
 
-You can see the full list of parameters in the `Zyte API Specification
-<https://docs.zyte.com/zyte-api/openapi.html#zyte-openapi-spec>`_.
-The ``url`` parameter is filled automatically from ``request.url``, other 
-parameters should be set explicitly.
+See the `Zyte API documentation`_ to learn about Zyte API parameters.
 
-The raw Zyte API response can be accessed via the ``raw_api_response``
-attribute of the response object.
+.. _Zyte API documentation: https://docs.zyte.com/zyte-api/get-started.html
```

Comment:
It seems that from the docs there are 2 sections which are relevant for scrapy-zyte-api users who want to know about Zyte API parameters:

* https://docs.zyte.com/zyte-api/usage/extract.html and
* https://docs.zyte.com/zyte-api/openapi.html